CHANGELOG.md

```diff
@@ -84,6 +86,7 @@ for further information about branching and tagging conventions.
   `creatorDetails` of approved reset request
 - Now using main signers account id instead of accounts account ID for
   setting pre-issuance asset signer
+- Update `@tokend/js-sdk` version
```

Comment:
you already have the "Now using @tokend/js-sdk@1.4.4" point. just bump version there

Reply:
ok